packages/design-system/src/components/InputField/InputField.md

```diff
@@ -27,20 +27,55 @@ onInputChange = (e) => {
   onChange={onInputChange}
 />
 ```
+
 ```js noeditor
-initialState = { value: 'Input Field text', error: 'Validation message here' };
+initialState = { value: "Input Field text", error: "Validation message here" };
 
 <ComponentHtmlMarkup>
   <InputField
     value={state.value}
-    labelText='Input Field label'
+    labelText="Input Field label"
     error={state.error}
-    id='input-field-example-1'
-    placeholder='Placeholder...'
-    description='Optional helper text goes here'
+    id="input-field-example-1"
+    placeholder="Placeholder..."
+    description="Optional helper text goes here"
     onChange={() => {}}
   />
-</ComponentHtmlMarkup>
+</ComponentHtmlMarkup>;
+```
+
+<h3>Input with chars left counter</h3>
+
+```js
+initialState = { value: '', error: null, chars: 5 };
+
+onInputChange = (e) => {
+  if (e.target.value.length > 5) {
+    setState({
+      value: e.target.value,
+      error: 'Validation message here',
+      chars: 0
+    });
+  } else {
+    setState({
+      value: e.target.value,
+      error: null,
+      chars: 5 - e.target.value.length
```

Comment:
I think the example is not working correctly. https://www.figma.com/file/KCYyNivnhN4wSDOM0Qjas4/KnowledgeBase-onboarding?node-id=845%3A6022 - here we have ie. 13/20 - that means that 13 characters were used. In your example it's the opposite - you are showing how many chars are left which doesn't feel intuitive imho